base_layer/contacts/src/contacts_service/error.rs

```diff
@@ -47,6 +48,12 @@ pub enum ContactsServiceError {
     ConnectivityError(#[from] ConnectivityError),
     #[error("Outbound comms error: `{0}`")]
     OutboundCommsError(#[from] DhtOutboundError),
+    #[error("Error parsing address: `{source}`")]
+    MessageParsingError { source: TariAddressError },
+    #[error("Error decoding message: `{0}`")]
+    IllFormedMessageError(#[from] prost::DecodeError),
```

Comment:
```suggestion
    MalformedMessageError(#[from] prost::DecodeError),
```
Nit: Illformed just reads difficult